scripts/genLUT_ycbcr.py

```diff
@@ -1,32 +1,55 @@
+import posixpath
+
+
+jpeg_mat = [1.0, 0.0, 1.402, 1.0, -0.34414, -0.71414, 1.0, 1.772, 0.0]
+
+
 def write_rh(l, typ, names, filename):
-    arr = []
-    for inner in range(len(l)):
-        arr.append(f"static constexpr {typ} const {names[inner]}[] = {{ {', '.join(map(str, l[inner]))} }};")
-        
-    with open("../include/seedimg-filters/rh/" + filename, "w") as f:
-        for s in arr:
-            f.write(s + "\n")
-
-#ycbcr_jpeg2rgb
-#this will generate a .rh (resource header) in the respective folder, which can be included:
-#it will have the nine lookup tables needed for this operation.
-#which are: the nth row of the jpeg matrix * y, cb, cr of the pixel.
+    array = []
+    for index, inner in enumerate(l):
+        array.append(
+            f"static constexpr {typ} const {names[index]}[] = {{ {', '.join(map(str, inner))} }};"
+        )
+
+    with open(posixpath.join("../include/seedimg-filters/rh", filename), "w") as file:
```

Comment:
Why specifically `posixpath` and not the host OS compatible path?

Reply:
YOU ARE CORRECT